qa/integration/services/dockerized/Dockerfile

```diff
@@ -16,6 +16,11 @@ USER tester
 ENV WORKDIR /home/tester
 WORKDIR $WORKDIR
 
+# Generate TLS certificates that can be used by services.
+ENV CERTIFICATES_DIR $WORKDIR/certificates
+RUN mkdir -p $CERTIFICATES_DIR && \
```

Comment:
@wainersm yea it can, already been there :D I'm all for your suggestion of just creating the certificate on the host and then passing it to whatever part of the tests needs it :) That should be safe imo. Let's do it that way around?